### DISCUSSION TITLE
unified_transformer里面用的DuConv数据集可以自己填写吗？

### DISCUSSION THREAD

**Discussion by jameszhengwuji** [PaddlePaddle/PaddleNLP#1423] [Q&A] [answered]
我看过了DuConv下载的电影对话数据集合，感觉可以直接改其中的内容来重新训练。现在发现程序还会下载一个对应的词汇表。这个词汇表是如何生成的？我按照我自己的数据随意生成一个，程序能运行吗？
多谢！

**Comment by ZeyuChen** [comment, marked as answer]
你可以试一下，应该是可以的。@jamezshengwuji
